ArcGISRuntimeSDKQt_QMLSamples/Maps/DisplayDeviceLocationWithNmeaDataSources/DisplayDeviceLocationWithNmeaDataSources.qml

```diff
@@ -44,69 +47,94 @@ Rectangle {
             }
         }
 
+        // Create the NemaLocationDataSource to push NMEA sentences to
         locationDisplay.dataSource: NmeaLocationDataSource {
             id: nmeaLocationDataSource
             receiverSpatialReference: SpatialReference { wkid: 4326 }
         }
-        locationDisplay.autoPanMode: Enums.LocationDisplayAutoPanModeRecenter;
+        locationDisplay.autoPanMode: Enums.LocationDisplayAutoPanModeRecenter
     }
 
     Button {
         id: button
         anchors.horizontalCenter: parent.horizontalCenter
-        y: 5
+        anchors.bottom: parent.bottom
+        anchors.margins: 25
         width: 200
-        property bool sampleStarted: false
-        text: sampleStarted ? "STOP" : "START"
+        text: nmeaSimulationActive ? "RESET" : "START"
         onClicked: {
-            sampleStarted = !sampleStarted;
-            if (sampleStarted) {
-                timer.start();
-                nmeaLocationDataSource.start();
-                mapView.locationDisplay.start();
-            }
-            else {
-                timer.stop();
-                timer.iterator = 0;
-                nmeaLocationDataSource.stop();
-                mapView.locationDisplay.stop();
-            }
+            nmeaSimulationActive = !nmeaSimulationActive;
+            if (nmeaSimulationActive)
+                startSimulation();
+            else
+                endSimulation();
         }
     }
 
-    property var mockNmeaData: []
-
     FileFolder {
         id: mockNmeaDataFile
-        path: ":/Samples/Maps/DisplayDeviceLocationWithNmeaDataSources/redlands.nmea"
+        path: "://Samples/Maps/DisplayDeviceLocationWithNmeaDataSources/redlands.nmea"
```

Comment:
is this change intentional?

Reply:
Good catch. It's not. It's left over from testing how I can include `qrc` as we discussed. It still runs, but it should be removed.